lib/gringotts/gateways/sagepay.ex

```diff
@@ -111,39 +103,70 @@ defmodule Gringotts.Gateways.Sagepay do
 
   alias Gringotts.{Money, CreditCard, Response}
   @url "https://pi-test.sagepay.com/api/v1/"
+
+  # SagePay supports payment only by Merchant Authorization Id and Merchant name
+  # by generating merchant session key and then providing card details generates
+  # card_identifier which is required in authorization.
+
   @doc """
   Performs a (pre) Authorize operation.
 
   The authorization validates the `card` details with the banking network,
   places a hold on the transaction `amount` in the customer’s issuing bank.
 
-  > ** You could perhaps:**
-  > 1. describe what are the important fields in the Response struct
-  > 2. mention what a merchant can do with these important fields (ex:
-  > `capture/3`, etc.)
+  SagePay returns an trancaction ID string which can be used to:
+  * `capture/3` an amount.
+  * `void/2` abort deferred transaction.
+  * `refund/3` an amount.
 
   ## Note
 
-  > If there's anything noteworthy about this operation, it comes here.
+  * The merchantSessionKey expires after 400 seconds and can only be used to create one successful card identifier. 
+  * It will also expire and be removed after 3 failed attempts to create a card identifier.
```

Comment:
1. Who will expire? the session key or the transaction ID?
2. It'll be great if you add a link to SagePay docs for this!